src/reducers/playerStatsReducer.js

```diff
@@ -0,0 +1,17 @@
+export default (state = [], action) => {
+    switch (action.type) {
+        case 'FETCH_PLAYERS':
+            return action.payload;
+        default:
+            return state;
+    }
+};
+
+export default (state=[], action) => {
```

Comment:
Formatting:

`state=[]` should be `state = []`

Reply:
I dont need this file, so im gonna kill bill it.